test/test_distributions.py

```diff
@@ -105,17 +115,18 @@
 
 
 class TestDistributions(TestCase):
-    def _set_rng_seed(self, seed=0):
-        torch.manual_seed(seed)
-        if TEST_NUMPY:
-            np.random.seed(seed)
+    def setUp(self):
+        set_rng_seed(0)
```

Comment:
I think I'd prefer it if the numpy seeding was moved to `test/common.py` (with appropriate conditionals for whether or not numpy is available); we have numpy tests that are not in `test_distributions.py`, but if the random seeding is done here then no other files can benefit. There is an added hazard here which is that if `setUp` in `TestCase` ever changes, the fact that we are not calling the superclass method means that `TestDistributions` will silently fail to get this behavior.

Reply:
Good points! I've moved `.setUp()` to the base class and moved `set_rng_seed()` to tests/common.py.